packages/rs-dpp/src/serialization/serialization_traits.rs

```diff
@@ -31,6 +31,21 @@ pub trait PlatformSerializable {
     }
 }
 
+pub trait ReducedPlatformSerializable {
+    type Error;
+    fn reduced_serialize_to_bytes(&self) -> Result<Vec<u8>, Self::Error>;
+}
+
+pub trait ReducedPlatformDeserializable {
+    type Error;
```

Comment:
Why do you define the associated type and don't use it? Why do you need these traits in the first place?